example/generate_elasticsearch_files.py

```diff
@@ -0,0 +1,4 @@
+from core.ElasticSearchBulkGenerator import ElasticSearchGenerator
+
+if __name__ == '__main__':
+    ElasticSearchGenerator.generate_elasticsearch_files(input_file='mii_core_data_set/ontology/backend.zip')
```

Comment:
@michael-82  consider using argsparse to make the script input zip file configurable